package.json

```diff
@@ -51,6 +49,10 @@
     "eslint-plugin-import": "^2.28.1",
     "eslint-plugin-mocha": "^10.1.0",
     "eslint-plugin-prettier": "^5.0.0",
+    "@semantic-release/exec": "^6.0.3",
+    "@semantic-release/git": "^10.0.1",
+    "@semantic-release/gitlab": "^12.0.2",
+    "semantic-release": "^20.1.3",
```

Comment:
Use semantic release version matching the one used in github action https://github.com/affinidi/passport-affinidi/blob/main/.github/workflows/github-release-workflow.yml#L34C18-L34C37